components/TARGET_PSA/services/crypto/COMPONENT_SPE/psa_crypto_partition.c

```diff
@@ -19,6 +19,14 @@
 // ------------------------- Globals ---------------------------
 static int psa_spm_init_refence_counter = 0;
 
+#define MAX_DATA_CHUNK_SIZE 400
```

Comment:
Consider making this configurable, maybe wrap it with an `ifndef`

Reply:
fixed